tests/stdlib/tsqlitebindatas.nim

```diff
@@ -48,3 +48,25 @@ block tsqlitebindatas: ## db_sqlite binary data
 
   db.close()
   doAssert tryRemoveFile(dbName)
+
+
+block:
+  block:
+    var db = db_sqlite.open("db.sqlite3", "", "", "")
+    try:
+      db.exec(sql("CREATE TABLE table1 (url TEXT, other_field INT);"))
+      db.exec(sql("REPLACE INTO table (url, another_field) VALUES (?, '123');"))
+    except DbError as e:
+      doAssert e.msg == "The number of parameter doesn't match that of \"?\" "
```

Comment:
this test doesn't actually ensure that the exception is raised; you need a "witness" variable to ensure this, called after the whole block
```nim
var witness = false
...
   witness = true

doAssert witness
```

Reply:
true